src/lib/api/UserAPI.ts

```diff
@@ -41,3 +55,113 @@ export const getUserByHandle = async (token: string, handle: string): Promise<Pu
 
   return response.data.user;
 };
+
+/**
+ * Update current user's profile picture
+ * @param token Authorization bearer token
+ * @param picture Profile picture
+ * @returns User's full profile
+ */
+export const uploadProfilePicture = async (
+  token: string,
+  picture: Blob
+): Promise<PrivateProfile> => {
+  const requestUrl = `${config.api.baseUrl}${config.api.endpoints.user.profilepicture}`;
+
+  const requestBody = new FormData();
+  requestBody.append('image', picture);
+
+  const response = await axios.post<UpdateProfilePictureResponse>(requestUrl, requestBody, {
+    headers: {
+      Authorization: `Bearer ${token}`,
+    },
+  });
+
+  return response.data.user;
+};
+
+/**
+ * Update current user's private profile
+ * @param token Authorization bearer token
+ * @param user Profile changes
+ * @returns User's full profile
+ */
+export const updateCurrentUser = async (
```

Comment:
Thoughts on renaming this function name to `updateCurrentUserProfile` or something? at a glance it looked like this function was switching which user was logged in